src/homework4.py

```diff
@@ -0,0 +1,421 @@
+import sys
+
+import numpy as np
+import pandas as pd
+import statsmodels.api as sm
+from plotly import express as px
+from plotly import figure_factory as ff
+from plotly import graph_objects as go
+from plotly.subplots import make_subplots
+from sklearn.ensemble import RandomForestClassifier
+
+
+def response_type_check(df, response):
+    # Determine if response is continuous or boolean
+    unique_responses = df[response].nunique()
+
+    if unique_responses == 2:
```

Comment:
I personally really like this line of code, very easy to understand and does the job perfectly!